src/main/java/module/transfer/history/TransferTableModel.java

```diff
@@ -1,213 +1,265 @@
-
-            case 6:
-
-                if (transfer.getType() == PlayerTransfer.BUY) {
-                    return transfer.getSellerName();
-                } else {
-                    return transfer.getBuyerName();
-                }
-
-            case 7:
-                return transfer.getPrice();
-
-            case 8:
-                return transfer.getTsi();
-
-            case 9:
-
-                if (transfer.getPlayerInfo() != null) {
-                    return transfer.getPlayerInfo().getLeadership();
-                } else {
-                    return -1;
-                }
-
-            case 10:
-
-                if (transfer.getPlayerInfo() != null) {
-                    return transfer.getPlayerInfo().getExperience();
-                } else {
-                    return -1;
-                }
-
-            case 11:
-
-                if (transfer.getPlayerInfo() != null) {
-                    return transfer.getPlayerInfo().getForm();
-                } else {
-                    return -1;
-                }
-
-            case 12:
-
-                if (transfer.getPlayerInfo() != null) {
-                    return transfer.getPlayerInfo().getStamina();
-                } else {
-                    return -1;
-                }
-
-            case 13:
-
-                if (transfer.getPlayerInfo() != null) {
-                    return transfer.getPlayerInfo().getGKskill();
-                } else {
-                    return -1;
-                }
-
-            case 14:
-
-                if (transfer.getPlayerInfo() != null) {
-                    return transfer.getPlayerInfo().getDEFskill();
-                } else {
-                    return -1;
-                }
-
-            case 15:
-
-                if (transfer.getPlayerInfo() != null) {
-                    return transfer.getPlayerInfo().getPMskill();
-                } else {
-                    return -1;
-                }
-
-            case 16:
-
-                if (transfer.getPlayerInfo() != null) {
-                    return transfer.getPlayerInfo().getPSskill();
-                } else {
-                    return -1;
-                }
-
-            case 17:
-
-                if (transfer.getPlayerInfo() != null) {
-                    return transfer.getPlayerInfo().getWIskill();
-                } else {
-                    return -1;
-                }
-
-            case 18:
-
-                if (transfer.getPlayerInfo() != null) {
-                    return transfer.getPlayerInfo().getSCskill();
-                } else {
-                    return -1;
-                }
-
-            case 19:
-
-                if (transfer.getPlayerInfo() != null) {
-                    return transfer.getPlayerInfo().getSPskill();
-                } else {
-                    return -1;
-                }
-
-            default:
-                return ""; //$NON-NLS-1$
+    @Override
+    protected void initData() {
```

Comment:
Interestingly I have been looking into migrating these this morning, and was really thinking I should look into removing all the duplication in that space.  On my todo!